examples/ansible/code/Synchronize_DNS_Zones

```diff
@@ -0,0 +1,50 @@
+- name: Sync DNS
+  hosts: <your Bind-Server>
+  tasks:
+    - name: get Managed Zones Names
+      ansible.builtin.set_fact:
+        zone_names: "{{ query('netbox.netbox.nb_lookup', 'zones', plugin='netbox_dns',
+                           api_endpoint='https://<nextcloud-url>/',
+                           api_filter='status=active status=deprecated',
```

Comment:
You can use `api_filter='active=true'` instead. That avoids potential problems with changes to the list of zone statuses considered active (which is in the queue).

Reply:
are the deprecated zones considered active aswell?

Reply:
No, currently only zones that are `active` are considered active. That's a use case for making the list of states considered active configurable - would you like to create an FR for that?